<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdDisplayDialog</class>
 <widget class="QDockWidget" name="QxrdDisplayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QCheckBox" name="m_InterpolatePixels">
      <property name="toolTip">
       <string>Gives a smoother display when zoomed in</string>
      </property>
      <property name="text">
       <string>Interpolate Pixel Values</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="m_MaintainAspectRatio">
      <property name="toolTip">
       <string>Try to maintain the aspect ratio of the image as it is zoomed and resized</string>
      </property>
      <property name="text">
       <string>Maintain Aspect Ratio</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Color Map</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="m_DisplayColorMap">
      <property name="toolTip">
       <string>Choose the color map for image display</string>
      </property>
      <item>
       <property name="text">
        <string>Grayscale</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Inverted grayscale</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Earth</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Spectrum</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fire</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ice</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="m_DisplayImage">
      <property name="toolTip">
       <string>Display image data</string>
      </property>
      <property name="text">
       <string>Display Image</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="m_DisplayMask">
      <property name="toolTip">
       <string>Display Mask</string>
      </property>
      <property name="text">
       <string>Display Mask</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="m_DisplayOverflow">
      <property name="toolTip">
       <string>Display Overflow</string>
      </property>
      <property name="text">
       <string>Display Overflow</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Overflow Level:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="m_OverflowLevel">
      <property name="toolTip">
       <string>overflow (saturation) level (0..65535)</string>
      </property>
      <property name="maximum">
       <number>65536</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QComboBox" name="m_DisplayScalingMode">
      <property name="toolTip">
       <string>Image display intensity scaling mode</string>
      </property>
      <item>
       <property name="text">
        <string>Display based on percentages of data range</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Display based on percentiles of pixel counts</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Display based on abs. values</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QStackedWidget" name="m_DisplayParmsStack">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_15">
        <item row="2" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Min (%):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="m_DisplayMinimumPct">
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="minimum">
           <double>-20.000000000000000</double>
          </property>
          <property name="maximum">
           <double>120.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Max (%):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="m_DisplayMaximumPct">
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="minimum">
           <double>-20.000000000000000</double>
          </property>
          <property name="maximum">
           <double>120.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="m_Display_5pct">
          <property name="text">
           <string>5% Range</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="m_Display_10pct">
          <property name="text">
           <string>10% Range</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="m_Display_100pct">
          <property name="text">
           <string>100% Range</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>88</width>
            <height>58</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_18">
        <item row="0" column="0">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Min (Percentile):</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="m_DisplayMinimumPctle">
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="minimum">
           <double>-20.000000000000000</double>
          </property>
          <property name="maximum">
           <double>120.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Max (Percentile):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="m_DisplayMaximumPctle">
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="minimum">
           <double>-20.000000000000000</double>
          </property>
          <property name="maximum">
           <double>120.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>70</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_19">
        <item row="0" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Minimum:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="m_DisplayMinimumVal">
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="minimum">
           <double>-1000000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Maximum:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="m_DisplayMaximumVal">
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="minimum">
           <double>-1000000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="m_AutoRange">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Auto Range</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <spacer name="verticalSpacer_10">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="m_DisplayOptionsButton">
      <property name="text">
       <string>More Options...</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="m_DisplayImageLog">
      <property name="toolTip">
       <string>Logarithmic Intensity Scaling</string>
      </property>
      <property name="text">
       <string>Log Scaling</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
